src/.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
+
+[run]
+source=./
+omit=*/tests/*
```

Comment:
You need to add `omit` in this config file and in the Makefile (https://github.com/marmelab/pentago-python/pull/12/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R13) ?

Reply:
That was a mistake :+1:  !